alluka/_types.py

```diff
@@ -256,6 +258,11 @@ def __init__(
         self.callback = callback
         self.type = type
 
+    def __getattr__(self, __name: str) -> Any:
```

Comment:
Can this be typed as returning typing.NoReturn

Reply:
Done, thanks!